### ISSUE TITLE
Please check the dialog when deleting the sprite.

### ISSUE THREAD

**Issue by digitarhythm** [LLK/scratch-gui#4311]
### Expected Behavior

When clicking the delete button attached to the sprite, a dialog for selecting "OK" or "NO" is displayed.
Because it erases it erroneously when selecting on the tablet.

### Actual Behavior

It is deleted without question.

### Steps to Reproduce

Press the delete sprite button.

### Operating System and Browser

ChromeOS 71

**Comment by jfo8000** [comment]
Yes this is a great idea!  Also I would like to be able to restore more than just the last sprite.  Had a kid delete the main sprite in my project...   would be nice if the deleted sprites went into a trash section of the backpack.

**Comment by apple502j** [comment]
The right-click menu is updated.

**Comment by carljbowman** [comment]
We will continue to monitor this issue. Originally we had a confirm browser modal, but we were getting feedback that folks were annoyed clicking the confirm. This was also right around the time we had added "Undo" (or in this case "Revert Sprite"), so we thought we'd try removing the confirm.

For now, it'd be great to try things with @apple502j's update to the context menu and keep an eye on things. If we keep seeing the problem, we could explore a two step delete process. Maybe a process that does not use a modal that takes focus, but is UI at the level of Sprite tile. 

**Quick rough sketches**
Clicking Delete
<img width="495" alt="screen shot 2019-01-17 at 8 52 59 am" src="https://user-images.githubusercontent.com/3409578/51323146-58fbda00-1a35-11e9-8ec6-9b4e6977f9f0.png">

Tile Level Confirmation
<img width="496" alt="screen shot 2019-01-17 at 8 53 09 am" src="https://user-images.githubusercontent.com/3409578/51323176-6d3fd700-1a35-11e9-87bf-77930a07dc36.png">

**Comment by jfo8000** [comment]
Love that you guys are thinking about it.  The in-tile confirm seems like a good balance to me at first blush...
